lib/xandra/connection.ex

```diff
@@ -9,20 +9,23 @@ defmodule Xandra.Connection do
   @default_timeout 5_000
   @default_socket_options [packet: :raw, mode: :binary, active: false]
 
-  defstruct [:socket, :prepared_cache, :compressor]
+  defstruct [:socket, :prepared_cache, :compressor, :atom_keys?]
 
   def connect(options) do
     address = Keyword.fetch!(options, :address)
     port = Keyword.fetch!(options, :port)
     prepared_cache = Keyword.fetch!(options, :prepared_cache)
     compressor = Keyword.get(options, :compressor)
+    atom_keys? = Keyword.get(options, :atom_keys?, false)
```

Comment:
Let's call the key `atom_keys`, not `atom_keys?`. The variable is fine with the question mark, but for options we tend not to use it for some reason.

Reply:
✅